suite-common/wallet-utils/src/ethereumStakingUtils.ts

```diff
@@ -0,0 +1,132 @@
+import { NetworkSymbol, getNetworkFeatures } from '@suite-common/wallet-config';
+import { isArrayMember } from '@trezor/utils';
+
+export const getEverstakePool = (account?: Account) => {
+    if (account?.networkType !== 'ethereum') {
+        return undefined;
+    }
+
+    return account?.misc?.stakingPools?.find(pool => pool.name === 'Everstake');
+};
+
+export const getAccountEverstakeStakingPool = (
+    account?: Account,
+): StakingPoolExtended | undefined => {
+    const pool = getEverstakePool(account);
+
+    if (!pool) return undefined;
+
+    return {
+        ...pool,
+        autocompoundBalance: fromWei(pool.autocompoundBalance, 'ether'),
+        claimableAmount: fromWei(pool.claimableAmount, 'ether'),
+        depositedBalance: fromWei(pool.depositedBalance, 'ether'),
+        pendingBalance: fromWei(pool.pendingBalance, 'ether'),
+        pendingDepositedBalance: fromWei(pool.pendingDepositedBalance, 'ether'),
+        restakedReward: fromWei(pool.restakedReward, 'ether'),
+        withdrawTotalAmount: fromWei(pool.withdrawTotalAmount, 'ether'),
+        totalPendingStakeBalance: fromWei(
+            new BigNumber(pool.pendingBalance).plus(pool.pendingDepositedBalance).toString(),
+            'ether',
+        ),
+        canClaim:
+            new BigNumber(pool.claimableAmount).gt(0) &&
+            new BigNumber(pool.withdrawTotalAmount).eq(pool.claimableAmount),
+    };
+};
+
+export const getAccountAutocompoundBalance = (account?: Account) => {
+    const pool = getAccountEverstakeStakingPool(account);
+
+    return pool?.autocompoundBalance ?? '0';
+};
+
+export const getAccountTotalStakingBalance = (account?: Account) => {
+    const pool = getAccountEverstakeStakingPool(account);
+
+    return new BigNumber(pool?.autocompoundBalance ?? '0')
+        .plus(pool?.pendingBalance ?? '0')
+        .plus(pool?.pendingDepositedBalance ?? '0')
+        .plus(pool?.withdrawTotalAmount ?? '0')
+        .toFixed();
+};
+
+export const getEthereumCryptoBalanceWithStaking = (account: Account) => {
+    const stakingBalance = getAccountTotalStakingBalance(account);
+
+    return new BigNumber(account.formattedBalance).plus(stakingBalance).toString();
+};
+
+export function isSupportedEthStakingNetworkSymbol(
+    networkSymbol: NetworkSymbol,
+): networkSymbol is SupportedNetworkSymbol {
+    return isArrayMember(networkSymbol, supportedNetworkSymbols);
+}
+
+export const getStakingSymbols = (networkSymbols: NetworkSymbol[]) =>
+    networkSymbols.reduce((acc, networkSymbol) => {
+        if (
+            isSupportedEthStakingNetworkSymbol(networkSymbol) &&
+            getNetworkFeatures(networkSymbol).includes('staking')
+        ) {
+            acc.push(networkSymbol);
+        }
+
+        return acc;
+    }, [] as SupportedNetworkSymbol[]);
+
+// Define signature constants
+const STAKE_SIGNATURE = '0x3a29dbae';
+const UNSTAKE_SIGNATURE = '0x76ec871c';
+const CLAIM_SIGNATURE = '0x33986ffa';
```

Comment:
Can those be added to your lib? and we just import it from @everstake/sdk?

Reply:
Agree, it could be added. But let's do this as a part of the separate PR. I remember we also had some hardcoded values in the blockbook that didn't belong to it.